src/qibolab/transpilers/star_connectivity.py

```diff
@@ -78,7 +78,7 @@ def is_satisfied(self, circuit):
         self.tlog("Circuit respects connectivity.")
         return True
 
-    def __call__(self, circuit, initial_layout):
+    def __call__(self, circuit, initial_layout={}):
```

Comment:
Do not use mutable objects as default parameters, as they shared between all the calls.

```py
    def __call__(self, circuit, initial_layout=None):
        initial_layout = initial_layout if initial_layout is not None else {}
```